versions.tf

```diff
@@ -18,6 +18,10 @@ terraform {
     local = {
       source = "hashicorp/local"
     }
+    google = {
```

Comment:
There should also be a .terraform.lock.hcl file change. We'll have to exclude that from .gitignore.

Reply:
Ah yes, good spot!